java/se/sics/mspsim/util/DotDiagram.java

```diff
@@ -169,7 +169,7 @@ public void setXLabel(String xLabel) {
     this.xLabel = xLabel;
   }
 
-  public void setName(int index, String name) {
+  public static void setName(int index, String name) {
```

Comment:
These methods have no meaning if they are static. Probably better to simply remove them since they are not fully implemented?

Reply:
Fixed.